documentation/modules/auxiliary/scanner/nessus/nessus_http_login.md

```diff
@@ -0,0 +1,85 @@
+## Vulnerable Application
+
+This module will attempt to authenticate to a Nessus server RPC interface.
+
+## Verification Steps
+1. Start msfconsole
+2. Do: `use auxiliary/scanner/nessus/nessus_rest_login`
```

Comment:
Is this meant to be `auxiliary/scanner/nessus/nessus_http_login`?

Reply:
yes. changed.